next.config.js

```diff
@@ -13,6 +13,9 @@ const nextConfig = {
   images: {
     domains: ['juicebox.money','jbx.mypinata.cloud'],
   },
+  typescript: {
+    ignoreBuildErrors: true,
```

Comment:
Can we remove this?

Reply:
yes, I removed it. It was added for quick deployment on vercel, but fixed the TS errors and now will build fine.